From: Andrej Shadura <andrewsh@debian.org>
Date: Sun, 2 Jan 2022 11:29:52 +0100
Subject: Default to the GTK interface, not Qt

---
 src/libaudcore/config.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libaudcore/config.cc b/src/libaudcore/config.cc
index d51bd32..b87bd84 100644
--- a/src/libaudcore/config.cc
+++ b/src/libaudcore/config.cc
@@ -42,7 +42,7 @@ static const char * const core_defaults[] = {
     "recurse_folders", "TRUE",
     "resume_playback_on_startup", "TRUE",
     "show_interface", "TRUE",
-    "use_qt", "TRUE",
+    "use_qt", "FALSE",
 
     /* equalizer */
     "eqpreset_default_file", "",
